fl4health/clients/basic_client.py

```diff
@@ -1241,35 +1166,6 @@ def update_before_epoch(self, epoch: int) -> None:
         """
         pass
 
-    def maybe_progress_bar(self, iterable: Iterable) -> Iterable:
```

Comment:
This is a pretty generic function. Moved it to utils/client.py. Let me know if you disagree with the change.